go/runtime/txpool/queues.go

```diff
@@ -0,0 +1,39 @@
+package txpool
+
+import (
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
+	"github.com/oasisprotocol/oasis-core/go/runtime/host/protocol"
+)
+
+type TxQueueMeta struct {
+	Raw  []byte
+	Hash hash.Hash
```

Comment:
Do you still plan to do this? It would make for a cleaner API preventing accidental modification of these immutable fields.

Reply:
ya sure I'll do that today

still have to be careful though, because most code that uses these are in the same package and has private member access

Reply:
changed

Reply:
Cool, just need to update the tests :-)

Reply:
![image](https://user-images.githubusercontent.com/38229038/186020056-65e1b2f4-3b3b-403f-8cd3-e1e7b55fa992.png)
at long last